jib-core/src/main/java/com/google/cloud/tools/jib/tar/TarStreamBuilder.java

```diff
@@ -71,10 +72,12 @@ public void addTarArchiveEntry(TarArchiveEntry entry) {
    *
    * @param contents the bytes to add to the tarball
    * @param name the name of the entry (i.e. filename)
+   * @param modificationTime the modification time of the entry
    */
-  public void addByteEntry(byte[] contents, String name) {
+  public void addByteEntry(byte[] contents, String name, Instant modificationTime) {
     TarArchiveEntry entry = new TarArchiveEntry(name);
     entry.setSize(contents.length);
+    entry.setModTime(modificationTime.getEpochSecond());
```

Comment:
Shouldn't this be `toEpochMillis()`? I guess writing a test will confirm it.

Reply:
Hi @chanseokoh the tests are in the unit test `TarStreamBuilderTest`
You are quite right it should be passed in using `toEpochMillis` because the implementation of TarArchiveEntry divides whats passed in by 1000.  I've updated the unit test to set an explicit time rather than the Epoch to catch that, and show that the expected time is `MODIFICATION_TIME.truncatedTo(SECONDS)`

Reply:
I don't see the changes. Forgot to push a new commit?